images/asciidoctor-pdf/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM registry.redhat.io/ubi8/ruby-25
+MAINTAINER CodeReady Containers <devtools-cdk@redhat.com>
+
+RUN gem install asciidoctor-pdf --pre
+
+VOLUME /docs
+WORKDIR /docs
+
```

Comment:
@praveenkumar i needed to add `USER root`, otherwise the file being created had weird permissions and ownership.

Reply:
Ack, I will add that.